nnsvs/model.py

```diff
@@ -46,7 +44,6 @@ def __init__(self, in_dim, hidden_dim, out_dim, num_layers=4, dropout=0.0):
         ]
 
         self.model = nn.Sequential(*model)
-
```

Comment:
Same as above

Reply:
I added new line there in the latest commit.